src/unit-taxonomy-tags-drawer/UnitTaxonomyTagsDrawer.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useMemo, useEffect } from 'react';
+import {
+  Container,
+  CloseButton,
+} from '@edx/paragon';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { useParams } from 'react-router-dom';
+import messages from './messages';
+import TaxonomyTagsCollapsible from './TaxonomyTagsCollapsible';
+import { extractOrgFromUnitId } from './utils';
+import {
+  useUnitTaxonomyTagsDataResponse,
+  useIsUnitTaxonomyTagsDataLoaded,
+  useUnitDataResponse,
+  useIsUnitDataLoaded,
+} from './api/hooks/selectors';
+import { useTaxonomyListDataResponse, useIsTaxonomyListDataLoaded } from '../taxonomy/api/hooks/selectors';
+
+const UnitTaxonomyTagsDrawer = ({ intl }) => {
```

Comment:
As mentioned above, this isn't specific to units, and I'm not a big fan of "Taxonomy" in the name, so I'd recommend a simpler name like `ContentTagsDrawer`